csvcubed-pmd/csvcubedpmd/csvwcli/__init__.py

```diff
@@ -4,7 +4,6 @@
 
 import csvcubedpmd.csvwcli.pull as pull
 
-
```

Comment:
I believe this extra line is required by the black formatter. Do you have it installed (and use it) in vscode?